hello.qmd

```diff
@@ -2,10 +2,44 @@
 
 ## What is Quarto?
 
+Quarto is an open-source scientific and technical publishing system built on Pandoc.
+It provides a unified authoring framework for data science, combining code, results, and narrative and outputs to HTML, PDF, Word, and more in a seamless manner.
+A few of the key features of Quarto that are applicable to Quarto users at any level are listed below:
+
+1.  **Markdown authoring**: Quarto documents are authored using **markdown**, a straightforward plain text format that allows you to add formatting elements to plain text documents using a simple syntax.
+
+2.  **Code integration**: Quarto extends the code capabilities of Pandoc by embedding code and output from various languages, including Python, R, Julia, and JavaScript.
+    This is made possible by execution engines like Jupyter, knitr, and Observable.
+
+3.  **Technical writing extensions**: Quarto includes extensions to Pandoc markdown specifically designed for technical writing that cover features like cross-references, sub-figures, layout panels, hoverable citations and footnotes, and more.
```

Comment:
Again if we want to de-emphasize Pandoc as a reference point maybe something like:
```suggestion
3.  **Technical writing extensions**: Quarto markdown adds syntax for technical writing that covers features like cross-references, sub-figures, layout panels, hoverable citations and footnotes, and more.
```